### ISSUE TITLE
Avali Mod Questions

### ISSUE THREAD

**Issue by BassManticore65** [Avali-Triage-Team/Avali#327]
This might not be too important, however, I remember in the development of the mod, an Avali drop ship with seats on the side and guns on the front. Also, is this mod still gaining content and content is being worked on, or is this mod not going to be updated anymore? Sorry for the questions, I just really like this mod, and I wanted to ask a few questions. Thanks.

**Comment by Fevix** [comment]
We're on hold right now, waiting for Ryuujin to finish the rewrite. 

Once that is done, yes, we'll be making content for this
